C/tests/c4Test.cc

```diff
@@ -203,22 +203,6 @@ _versioning(kC4RevisionTrees)
 {
     static once_flag once;
     call_once(once, [] {
-        c4log_enableFatalExceptionBacktrace();
```

Comment:
Did you mean to get rid of this?  Without it there is no backtrace on crash.

Reply:
That's in QuietReporter.cc now, along with all the logging setup.

Reply:
I noticed that as well but what about the list reporter?

Reply:
I figured this one obsoletes it, except as the superclass. It's a bit awkward having the logging initialization go here, but it needs to so that it can set the log levels and find the log files.